theme/brb/views/dataprivacy/privacy_anschrift.hbs

```diff
@@ -0,0 +1,6 @@
+<p>
```

Comment:
here the address of the main theme should be used instead!

Reply:
No it's not. There's 1 sentence missing about cookies used for user behaviour tracking that we do in main.

Reply:
I just follow given templates: https://ticketsystem.schul-cloud.org/browse/BRB-9
There, the text is different compared to main.